static/js/controllers/reports-content.js

```diff
@@ -45,10 +47,34 @@
         setMessageState(group, 'muted', 'scheduled');
       };
 
+      $scope.toggleExpand = function(selection) {
+        if ($scope.selectMode) {
+          selection.expanded = !selection.expanded;
+        }
+      };
+
+      $scope.deselect = function(selection, $event) {
+        if ($scope.selectMode) {
+          $event.stopPropagation();
+          for (var i = 0; i < $scope.selected.length; i++) {
+            if ($scope.selected[i].report._id === selection._id) {
+              $scope.selected.splice(i, 1);
+              $('#reports-list li[data-record-id="' + selection._id + '"] input[type="checkbox"]')
+                .prop('checked', false);
+              return;
+            }
+          }
+        }
+      };
+
       Changes({
         key: 'reports-content',
         filter: function(change) {
-          return $scope.selected && $scope.selected._id === change.id;
+          return $scope.selected &&
```

Comment:
General thought : is there any time where `$scope.selected` should be null? Now that it's a list, it should be empty but not null, right? So I don't think we want that case to fail silently... Here and kindof everywhere. 
It could be best to test for `$scope.selected.length` only, and crash if there's no `selected`, and we'll find bugs that way.